package.json

```diff
@@ -14,57 +14,75 @@
     "start": "ember serve",
-    "test": "npm-run-all lint:* test:*",
+    "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
     "@ember/test-waiters": "^3.0.1",
-    "ember-cli-babel": "^7.22.1",
-    "ember-cli-htmlbars": "^5.3.1",
-    "ember-modifier": "^2.1.1"
+    "ember-auto-import": "^2.6.1",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-htmlbars": "^6.2.0",
+    "ember-modifier": "^3.0.0 || ^4.0.0"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
-    "@glimmer/component": "^1.0.2",
-    "@glimmer/tracking": "^1.0.2",
+    "@ember/string": "^3.0.1",
+    "@ember/test-helpers": "^2.9.3",
+    "@embroider/test-setup": "^2.1.1",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-auto-import": "^1.6.0",
-    "ember-cli": "~3.22.0",
-    "ember-cli-dependency-checker": "^3.2.0",
+    "concurrently": "^7.6.0",
+    "ember-cli": "~4.11.0",
+    "ember-cli-dependency-checker": "^3.3.1",
     "ember-cli-deploy": "^1.0.2",
     "ember-cli-deploy-build": "^2.0.0",
     "ember-cli-deploy-git": "^1.3.4",
-    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.0",
+    "ember-cli-terser": "^4.0.2",
     "ember-code-snippet": "^3.0.0",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.1",
-    "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.6.0",
-    "ember-resolver": "^8.0.2",
-    "ember-source": "~3.22.0",
+    "ember-load-initializers": "^2.1.2",
+    "ember-page-title": "^7.0.0",
+    "ember-qunit": "^6.2.0",
+    "ember-resolver": "^10.0.0",
+    "ember-sinon-qunit": "^7.1.4",
+    "ember-source": "~4.11.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^2.14.0",
-    "ember-try": "^1.4.0",
-    "eslint": "^7.11.0",
-    "eslint-plugin-ember": "^9.3.0",
-    "eslint-plugin-node": "^11.1.0",
+    "ember-template-lint": "^5.6.0",
+    "ember-try": "^2.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-ember": "^11.4.7",
+    "eslint-plugin-n": "^15.6.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-qunit": "^7.3.4",
     "highlight.js": "^11.4.0",
     "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "qunit-dom": "^1.5.0",
-    "sinon": "^9.2.3"
+    "prettier": "^2.8.4",
+    "qunit": "^2.19.4",
+    "qunit-dom": "^2.0.0",
+    "sinon": "^15.0.2",
+    "webpack": "^5.75.0"
+  },
+  "peerDependencies": {
+    "ember-source": "^3.28.0 || ^4.0.0"
   },
   "engines": {
-    "node": "10.* || >= 12"
+    "node": "14.* || 16.* || >= 18"
+  },
+  "volta": {
```

Comment:
This has no impact for non-volta users, but it allows you to ensure local development uses the right versions of node and yarn. Otherwise, your lockfile could get overwritten by devs that have yarn 3 installed, thus breaking your CI builds using yarn 1.

More info: https://volta.sh/